### ISSUE TITLE
Expo Support

### ISSUE THREAD

**Issue by shamxeed** [flyerhq/react-native-chat-ui#25]
Firstly I will like to say a big thank you for this amazing and wonderful project that will soon overtake most of the unmaintained libraries out there. As I'm currently in need of something similar to this and came across your library but had one question, **does it support Expo Platform? **  Because my current development setup uses Expo.

Thank you so much once again!

**Comment by demchenkoalex** [comment]
Hi @shamxeed thanks for the kind words! :)

We have almost 0 dependencies and AFAIK `react-native-safe-area-context` (our one and only dependency with a native code) is available on Expo. Personally, I didn't try to run this on Expo but I don't see any reason for it to not run. It would be awesome if you could check whether this library works on Expo :)

**Comment by shamxeed** [comment]
Thanks, @demchenkoalex  for the fast response!!
I did give it a try and I'm happy to say it does support Expo out of the box but just that I'm experiencing some kinda issues, like:
1. The chat thread is not auto-scrolling when a new message is added, I've even created an issue with regard to that [here](https://github.com/flyerhq/react-native-chat-ui/issues/26#issue-905835832)
2. When I tried to align messages to the top by using the `flatListProps` prop is not actually aligning.
e.g: 
```jsx
  <Chat 
     ...
     flatListProps={{inverted: true}}
  />
```
3.  Finally, the `onEndReached` callback only fires once, that is when the screen mounted

```jsx
<Chat 
   ...
   flatListProps={{
      inverted: true, 
      onEndReached={() => console.log('List ended here...')}
   }}
/>
```

Once again thank you guys for this!!

**Comment by demchenkoalex** [comment]
2. I don't think it will be possible, because we do everything for the inverted list (I guess the inverted prop is always overwritten). We do not plan to support not inverted mode, because we don't know how our design will react. Right now we are focused on delivering v1.1 for Flutter (https://github.com/flyerhq/flutter_chat_ui/issues/24) you can check that issue to see what changes are coming, because the same will happen on React Native.

3. Pagination and `onEndReached` are actually on a branch right now and will be merged and delivered after we finish v1.1 for Flutter (will be v1.1 for React Native)

Regarding the other issue we will check that ASAP.

Thanks!
